lib/verdict.rb

```diff
@@ -20,7 +20,11 @@ def repository
   end
 
   def discovery
-    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| require f } if @directory
+    Dir[File.join(Verdict.directory, '**', '*.rb')].each { |f| load f } if @directory
```

Comment:
No, it's supposed to be able to be used with any Ruby application, but we provide a Railtie for integration with Rails.